### PR TITLE
Update frontend image

### PR DESCRIPTION
**Pull request by blackzlq**
1. Remove enabled filter for two filters.
2. Add more description on the filters

Tested-by: zlq

### CODE REVIEW

File: applications/rag/frontend/container/templates/index.html [hunk 1]
```diff
@@ -35,36 +35,40 @@
...
+        </select>
+        <p id="deidentify-template-msg" style="display: none;">Please go to your project to create the de-identification template.</p>
+      </div>
     </div>
 
     <!-- Checkbox to toggle nlp-filter-section -->
-    <div id="toggle-nlp-filter-section-div" style="display: none;">
-      <label for="toggle-nlp-filter-section">Show NLP Filter Range:</label>
+    <div id="toggle-nlp-filter-section-div">
+      <label for="toggle-nlp-filter-section"><a href="https://cloud.google.com/natural-language/docs/moderating-text">Text moderation</a>  analyzes a document against a list of safety attributes, which include "harmful categories" and topics that may be considered sensitive.:</label>
```
Comment (imreddy13) [inline applications/rag/frontend/container/templates/index.html:65]: Can we delete the colon at the end?

**Review by imreddy13 — APPROVED** [review]
thank you!

File: applications/rag/frontend/container/static/script.js [hunk 2]
```diff
@@ -48,27 +48,19 @@ function onReady() {
         enableForm(false);
 
         // Collect filter data
-        var filterEnabled = document.getElementById("toggle-filters").checked;
-        var body = JSON.stringify({
+        let data = {
             prompt: prompt,
```
Comment (andrewsykim) [inline applications/rag/frontend/container/static/script.js:52]: @blackzlq I'm adding some prompts to test in here https://github.com/GoogleCloudPlatform/ai-on-gke/pull/511 

Can you suggest some ways to test prompts with DLP/NLP?

Comment (blackzlq) [reply applications/rag/frontend/container/static/script.js:52]: Yeah, @imreddy13 sends the doc in chat room.

File: applications/rag/frontend/container/templates/index.html (hunk 1 above)
Comment (blackzlq) [reply applications/rag/frontend/container/templates/index.html]: Good catch!

File: applications/rag/frontend/container/static/script.js (hunk 2 above)
Comment (blackzlq) [reply applications/rag/frontend/container/static/script.js:52]: who worked with Robert De Niro and name one film they collaborated? with name filter to select the info_type personal_name and de-identity template to replace the content with info_type name. is good for DLP

and any prompt then set the range bar to 100 should return `The responses is deemed inapproriate for display`

`Which movie will show blowing up a building ` with range value 60 should got the `The responses is deemed inapproriate for display` and if we lower the value it should return the original response.

**Comment by blackzlq** [comment]
There is no way for this change affect existing test, we haven't update the image SHA, rerun the test

**Pull request merged**